src/client.ts

```diff
@@ -181,3 +179,5 @@ export class Client {
     return this.executeRequestOperation(operation);
   };
 }
+
+export const createClient = (opts: ClientOptions) => new Client(opts);
```

Comment:
Reordered this to the bottom so that `Client` isn't invoked before its definition (lint flagged this). Happy to return to the original and just disable lint for this.

Reply:
No, that's alright; we have really inconsistent ordering. @andyrichardson prefers define-after-use and ordering by importance, while I prefer a strict define-before-use. Any rule that enforces this would be a win, so it stays consistent across the codebase :+1: